src/patterns/components/links/react/info/inline.hbs

```diff
@@ -0,0 +1,120 @@
+<p class="sprk-u-mbm">
+  See below for available customization options:
+</p>
+
+<table class="drizzle-c-Table drizzle-c-Table--wide sprk-u-mbm">
+  <colgroup>
+    <col class="sprk-u-Width-40">
+    <col>
+  </colgroup>
+
+  <thead>
+    <tr>
+      <th>Prop</th>
+      <th>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        element
+      </td>
+
+      <td>
+        string, function
+      </td>
+
+      <td>
+        Can pass a router link or anchor tag. Defaults to anchor tag if no value is passed.
+      </td>
+    </tr>
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        variant
+      </td>
+
+      <td>
+        string
+      </td>
+
+      <td>
+        Can be 'simple', 'has-icon', 'plain', 'disabled', or 'unstyled'. Will cause the appropriate variant type to render.
+      </td>
+    </tr>
+
+    <tr>
+      <td class="sprk-u-FontWeight--bold">
+        href
+      </td>
+
+      <td>
+        string
+      </td>
+
+      <td>
+        The URL of the link. Defaults to '#' and preventDefault() if no href is provided.
```

Comment:
@guern1kn I know SprkLink got refactored a billion times, but, I think we realized the last refactor that the preventDefault() actually doesn't run when they provide an optional onClick prop. If that's the case, then this documentation should be updated. Can you take a look?